examples/minimal-cuda-solver/CMakeLists.txt

```diff
@@ -1,7 +1,15 @@
-set(target_name "minimal-cuda-solver")
-add_executable(${target_name} ${target_name}.cpp)
-target_link_libraries(${target_name} Ginkgo::ginkgo)
-target_include_directories(${target_name} PRIVATE ${PROJECT_SOURCE_DIR})
+cmake_minimum_required(VERSION 3.9)
+project(minimal-cuda-solver)
+
+# We only need to find Ginkgo if we build this example stand-alone
+if (NOT GINKGO_BUILD_EXAMPLES)
+    find_package(Ginkgo 1.4.0 REQUIRED)
```

Comment:
Don't we need to make sure we have `GINKGO_BUILD_CUDA`? Or we just let it fail if that's not the case?

Reply:
I would just let it fail in this case, since CUDA is not a compile-time dependency.